spec/runtime/executable_spec.rb

```diff
@@ -8,142 +8,9 @@
     G
   end
 
```

Comment:
van you make sure `bundle binstubs` is properly tested, and if not, change these specs to test it?